networks/p2p/server.go

```diff
@@ -56,9 +56,12 @@ const (
 	frameWriteTimeout = 20 * time.Second
 )
 
-// TODO-Klaytn-Node Below constants are duplicated with node packages. Use it temporarily until fix cycle import problem.
+type ConnType int
+
+const ConnTypeUndefined ConnType = -1
+
 const (
-	CONSENSUSNODE = iota
+	CONSENSUSNODE ConnType = iota
```

Comment:
Is there any reason to merge in p2p package?
The names of those and the way they are used are closer to the node.

Reply:
@jimni1222 This is to avoid cyclic-dependency problem, as the `node` package imports `p2p` package. Maybe we can move these to `common` package. How do you think? cc @aidan-kwon @nogang @KimKyungup

Reply:
If we can't move those into the node package, moving to the common package seems also a good idea to me.

Reply:
It sounds good to me moving them to `common` package.

Reply:
There are some code using node type in `p2p` package. and `node` package imports `p2p` package.
So I think `node` package can use the node type in `p2p`.
When @bitaholic implemetated discovery feature, I think he didn't want to change `node` package much.
Therefore he just wrote temporary node type in `p2p`.
Now we can merge these into `p2p`.

Reply:
@jimni1222 @aidan-kwon Thanks for your feedback =) I will update the PR.